util/build_docs.py

```diff
@@ -404,7 +404,7 @@ def install_hugo(install_dir):
     return hugo_bin_path
 
 
-def invoke_hugo(preview, hugo_bin_path):
+def invoke_hugo(preview, hugo_bin_path, bind_wan):
```

Comment:
Nit: Maybe it would be better to switch around the first or last two arguments? I think that `preview` and `bind_wan` are similar to each other, and different from `hugo_bin_path`. So I think this should be:
```python3
 def invoke_hugo(preview, bind_wan, hugo_bin_path):
```
or the other way around.

Reply:
Hi @rswarbrick, I added one more arg in the last commit, but then I put them altogether and put the bin path last.